### DISCUSSION TITLE
Community Meeting 2023 June 02

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#300] [Announcements]
Our next meeting will take place 4pm-5pm Eastern on June 2.

Agenda:

- maintainers adding themselves to list of watchers
- re-discuss the extent to which we would want or not want to remore redundant theorems
- trimming of commit messages when approving PRs with a squash merge
- guidelines on amount of detail to put in README files for properties and for spaces
  (see discussion in [Add P-space property #310](https://github.com/pi-base/data/pull/310) for example)
- Acceptance of correct but improvable PRs
- guidelines on amount of detail to put in README files for properties and for spaces
(see discussion in https://github.com/pi-base/data/pull/310 for example)
- "Hackathons", workhops, REUs?
- Preview: https://topology.pages.dev/

Please feel free to discuss/suggest more items below.

**Comment by StevenClontz** [comment]
I've updated the title with the date for our meeting. Please reply if you see this so I know this info is getting out to the community. :-)

**Reply by StevenClontz** [reply]
@ccaruvana @lyengulalp @hpecora1 testing to see if you get notified if you're tagged...

**Reply by ccaruvana** [reply]
I got notified via email with this, but not before this. I guess one needs to add the "All Activity" option under "Watch" to get notified other than mentions and threads you have posted on.

**Reply by StevenClontz** [reply]
"Be sure to subscribe, and hit that bell for notifications so you don't miss a video!" er post

**Comment by lyengulalp** [comment]
Yes.

**Reply by StevenClontz** [reply]
Just so I know who's hearing about stuff, did you get a notification before I tagged you?

**Comment by lyengulalp** [comment]
Same here.

**Reply by StevenClontz** [reply]
Thanks - I'm exploring GitHub discussion boards for organizing a few different communities, so this feedback is good.

**Comment by hpecora1** [comment]
Bit late to the party, but I had the same experience as Chris.

**Comment by prabau** [comment]
Some suggestions for items on the agenda:
- re-discuss the extent to which we would want or not want to remore redundant theorems
- trimming of commit messages when approving PRs with a squash merge
- maintainers adding themselves to list of watchers

**Comment by prabau** [comment]
One more suggestion for the agenda:
- guidelines on amount of detail to put in README files for properties and for spaces
(see discussion in #310 for example)

**Comment by prabau** [comment]
One more: generic examples of spaces (not fully determined up to homeomorphism).  See discussion in #314.

**Comment by StevenClontz** [comment]
FYI I'm running late. I'm hoping to join the call 4:15 Eastern

**Comment by StevenClontz** [comment]
Y'all should feel free to start discussing things without me. Patrick suggested a few of the items so you can lead those discussions without me if you like.
